src/main/java/edu/ucla/library/iiif/fester/verticles/S3BucketVerticle.java

```diff
@@ -59,20 +83,24 @@ public void start() throws Exception {
             final String s3AccessKey = config.getString(Config.S3_ACCESS_KEY);
             final String s3SecretKey = config.getString(Config.S3_SECRET_KEY);
             final String s3RegionName = config.getString(Config.S3_REGION, "us-east-1");
-            final String endpoint = config.getString(Config.S3_ENDPOINT);
+            final S3ClientOptions opts = new S3ClientOptions().setCredentials(s3AccessKey, s3SecretKey);
 
-            // Check to see that we're not overriding the default S3 endpoint
-            if (endpoint == null || Constants.S3_ENDPOINT.equals(endpoint)) {
-                final String regionEndpoint = RegionUtils.getRegion(s3RegionName).getServiceEndpoint("s3");
+            if (RegionUtils.getRegion(s3RegionName) != null) {
```

Comment:
What would be an example value for `s3RegionName` that would cause this conditional to be false?

Reply:
"blah-blah-region" ... or any non-valid S3 region that their provider doesn't know.